content/docs/introduction/regions.md

```diff
@@ -20,6 +20,14 @@ Neon currently supports the following AWS regions:
 - Asia Pacific (Singapore) &mdash; `aws-ap-southeast-1`
 - Asia Pacific (Sydney) &mdash; `aws-ap-southeast-2`
 
+## Azure regions
+
+- East US 2 region &mdash; `azure-us-east-2`
```

Comment:
I suggest the region name be formatted in the same way as the Azure CLI. `azure-eastus2` as this would be the most natural to users.

Reply:
Thank you. Fixed.